### PR TITLE
workflow updates 1.1

### PR DESCRIPTION
**Pull request by soujay**
workflow updates 1.1

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,10 +11,6 @@ env:
   DOTNET_NOLOGO: true
 jobs:
   setup:
-    if:
```
Comment (DavidBoike) [inline .github/workflows/ci.yml:14]: It’s outside the PR so I can’t comment on it but the pull_request_target trigger needs to come out.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -154,33 +150,31 @@ jobs:
       fail-fast: false
     steps:
       - name: Checkout
-        if: github.event_name != 'pull_request_target'
-        uses: actions/checkout@v2.4.0
+        uses: actions/checkout@v3.0.2
         with:
           fetch-depth: 0
       - name: Checkout for Dependabot
```
Comment (DavidBoike) [inline .github/workflows/ci.yml:156]: This whole step needs to come out

**Review by DavidBoike — DISMISSED** [review]
Some pull_request_target related stuff

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -24,9 +20,9 @@ jobs:
         env:
           SECRETS_AVAILABLE: ${{ secrets.SECRETS_AVAILABLE }}
         shell: pwsh
-        run: exit $(If ($env:SECRETS_AVAILABLE -eq 'true') { 0 } Else { 1 })    
+        run: exit $(If ($env:SECRETS_AVAILABLE -eq 'true') { 0 } Else { 1 })
       - name: Azure login
-        uses: azure/login@v1.4.2
+        uses: azure/login@v1
```
Comment (ramonsmits) [inline .github/workflows/ci.yml]:
```suggestion
        uses: azure/login@v1.4.5
```

**Review by ramonsmits — APPROVED** [review]

**Review by DavidBoike — APPROVED** [review]

**Pull request merged**